src/status.rs

```diff
@@ -49,6 +49,7 @@ pub struct State {
 #[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
 pub struct Event {
     pub name: String,
+    #[serde(rename = "type")]
     pub _type: String,
```

Comment:
Is this the correct style to use to avoid naming conflicts, or would `type_` be better?

Otherwise good to merge.